lib/utils.pm

```diff
@@ -720,7 +720,7 @@ that the boot partition is encrypted.
 
 =cut
 sub is_boot_encrypted {
-    return 0 if get_var('UNENCRYPTED_BOOT');
+    return 0 if get_var('UNENCRYPTED_BOOT') || get_var('HOME_ENCRYPTED');
```

Comment:
`HOME_ENCRYPTED` var is not used in your yaml scheduling file.

I guess, you can remove it, just set the existing `UNENCRIPTED_BOOT` in scheduling file.

Reply:
yes  can replace the Variable. but i have set it up in the TestSuite. Should it be in yaml as well?

Reply:
You have both `ENCRYPTED` and `HOME_ENCRYPTED` variables in the test suite. 

It seems like `HOME_ENCRYPTED` only needed to return `0` in `boot_encrypted` function. Why not just set already existing variable `UNENCRYPTED_BOOT=1` instead of introducing new one for the same reason?

If you want to keep `HOME_ENCRYPTED`, then the logic here looks questionable for me. How "home" encryption related to the "boot" encryption? May it be the case, when both "boot" and "home" are encrypted?

Reply:
i have remove it and use the `UNENCRYPTED_BOOT`. it is set in TestSuite